compiler/rustc_mir/src/dataflow/impls/storage_liveness.rs

```diff
@@ -150,6 +150,8 @@ impl<'mir, 'tcx> dataflow::GenKillAnalysis<'tcx> for MaybeRequiresStorage<'mir,
             | StatementKind::Nop
             | StatementKind::Retag(..)
             | StatementKind::StorageLive(..) => {}
+
+            StatementKind::CopyNonOverlapping(..) => todo!(),
```

Comment:
Since this just uses pointers, its places were already given storage at the start of this function, so you can safely add it to the nop-list above